backstory/src/main/java/com/google/sps/story/PromptManagerAPIsClient.java

```diff
@@ -25,33 +25,29 @@
  * Facilitates API calls through a single object.
  */
 public class PromptManagerAPIsClient {
-  /** Client object for word classification */
-  private NLServiceClient wordClassifier;
   /** Client object for word fetching */
   private DatamuseRequestClient wordFetcher;
 
   /**
-   * Initialize API objects.
-   *
-   * @throws IOException Exception for network problem.
+   * Initialize Datamuse API object.
    */
   public PromptManagerAPIsClient() throws IOException {
-    try {
-      // Instantiate API objects
-      wordClassifier = new NLServiceClient();
-      wordFetcher = new DatamuseRequestClient();
-    } catch (IOException ioException) {
-      throw ioException;
-    }
+    wordFetcher = new DatamuseRequestClient();
   }
 
   /**
    * Classifies an input of strings based on their part of speech.
    *
    * @param words A list of Strings containing keywords for prompts.
    * @return A mapping of WordTypes to given words.
+   * @throws IOException Exception for network problem.
    */
-  public Map<WordType, List<String>> groupByWordType(List<String> words) {
+  public Map<WordType, List<String>> groupByWordType(List<String> words) throws IOException {
+    NLServiceClient wordClassifier = new NLServiceClient();
```

Comment:
Can this be named nlServiceClient so that the type of the variable is clearer?

Reply:
I changed this. Thanks!

Reply:
Done